flink-ml-servable-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionModelServable.java

```diff
@@ -0,0 +1,148 @@
+
+        input.addColumn(getPredictionCol(), DataTypes.DOUBLE, predictionResults);
+        input.addColumn(
+                getRawPredictionCol(), DataTypes.VECTOR(BasicType.DOUBLE), rawPredictionResults);
+
+        return input;
+    }
+
+    public LogisticRegressionModelServable setModelData(InputStream... modelDataInputs)
+            throws IOException {
+        Preconditions.checkArgument(modelDataInputs.length == 1);
+
+        DataInputViewStreamWrapper inputViewStreamWrapper =
+                new DataInputViewStreamWrapper(modelDataInputs[0]);
+
+        DenseVectorSerializer serializer = new DenseVectorSerializer();
+        coefficient = serializer.deserialize(inputViewStreamWrapper);
+
+        return this;
+    }
+
+    public static LogisticRegressionModelServable load(String path) throws IOException {
+        LogisticRegressionModelServable servable =
+                ServableReadWriteUtils.loadServableParam(
+                        path, LogisticRegressionModelServable.class);
+
+        try (InputStream fsDataInputStream = ServableReadWriteUtils.loadModelData(path)) {
+            DataInputViewStreamWrapper dataInputViewStreamWrapper =
+                    new DataInputViewStreamWrapper(fsDataInputStream);
+            DenseVectorSerializer serializer = new DenseVectorSerializer();
+            servable.coefficient = serializer.deserialize(dataInputViewStreamWrapper);
+            return servable;
+        }
+    }
+
+    /**
+     * The main logic that predicts one input data point.
+     *
+     * @param feature The input feature.
+     * @param coefficient The model parameters.
+     * @return The prediction label and the raw probabilities.
+     */
+    public static Tuple2<Double, DenseVector> predictOneDataPoint(
+            Vector feature, DenseVector coefficient) {
+        double dotValue = BLAS.dot(feature, coefficient);
+        double prob = 1 - 1.0 / (1.0 + Math.exp(dotValue));
+        return Tuple2.of(dotValue >= 0 ? 1. : 0., Vectors.dense(1 - prob, prob));
+    }
+
+    /**
+     * Serializes the model data into byte array which can be saved to external storage and then be
+     * used to update the Servable by `TransformerServable::setModelData` method.
+     *
+     * @param modelData The model data to be serialized.
+     * @return The serialized model data in byte array.
+     */
+    public static byte[] serialize(DenseVector modelData) throws IOException {
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+
+        DataOutputViewStreamWrapper outputViewStreamWrapper =
+                new DataOutputViewStreamWrapper(outputStream);
+
+        DenseVectorSerializer serializer = new DenseVectorSerializer();
+        serializer.serialize(modelData, outputViewStreamWrapper);
+
+        return outputStream.toByteArray();
+    }
+
+    @Override
+    public Map<Param<?>, Object> getParamMap() {
+        return paramMap;
+    }
+
+    public DenseVector getCoefficient() {
```

Comment:
Is it necessary to add this public API for users to get servable model data? This public API was not discussed in any existing FLIP.